doc/test-framework/writing-e2e-tests.md

```diff
@@ -0,0 +1,238 @@
+# Using the Operator SDK's Test Framework to Write E2E Tests
+
+End-to-end tests are an essential tool to ensure that an operator works
```

Comment:
I think `End-to-end test` is a just type of test not really an tool. Maybe just saying `End-to-end tests are essential to ensure`  is enough?